docs/sphinx/source/whatsnew/v0.8.1.rst

```diff
@@ -9,7 +9,7 @@ Breaking changes
 
 Deprecations
 ~~~~~~~~~~~~
-
+* :py:func:`pvlib.irradiance.liujordan` is deprecated.
```

Comment:
these links will break and cause documentation build warnings when we remove the function, so I'd rather use double ticks now